### DISCUSSION TITLE
New video about building a Quasar v2 Notes app

### DISCUSSION THREAD

**Discussion by smolinari** [quasarframework/quasar#8711] [Show and Tell]
Justin Brooks(@jsbroks), a member of the Quasar Community of professional developers, has also brought out a Video on building a Notes app with Quasar. He also goes over VueJS composition API. 

Awesome! 👍🔥💯😍

https://www.youtube.com/watch?v=qPkSwo8QyoA

**Comment by armenr** [comment]
Honestly, this video NAILS it from the start! One of the most useful and convenient aspects of Quasar is the layoutbuilder and how quickly you can specify an entire responsible layout with just a combination of letters :).

This is one of the most underrated and least talked-about aspects of Quasar in my opinion. If nothing else, this right here was a huge reason I chose to go with Quasar instead of prime or vuetify.
